.github/workflows/test-on-pr.yml

```diff
@@ -9,7 +9,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [ubuntu-latest, macos-latest, windows-latest]
+        platform: [macos-latest, windows-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        fetch-depth: 3
+    - name: install stable
+      uses: actions-rs/toolchain@v1
+      with:
+          toolchain: stable
+    - name: build
+      run: |
+           cd ./tauri
+           cargo build
+
+  build-tauri-ubuntu-latest:
```

Comment:
GitHub only allows alphanumeric characters in job names, so I wasn't able to name this job `build-tauri (ubuntu-latest)` so that it would look like the `build-tauri` jobs on other platforms